frontend/src/services/http/service-docs.tsx

```diff
@@ -57,12 +62,49 @@ function useServiceDocsHttpService(): ServiceDocsHttpService {
     );
   }
 
-  return {
-    listAllServiceDocs: (): Promise<
-      ListAllServiceDocsHttpResponse | UnknownHttpError
-    > => {
-      return listAllServiceDocs();
+  async function deleteSingleServiceDoc(
+    serviceName: string,
+  ): Promise<DeleteSingleServiceDocResponse | UnknownHttpError> {
+    const response = await httpService.performRegularApiRequest(
+      (configuration) =>
+        doDeleteSingleApiKey(configuration, { serviceName: serviceName }),
+    );
+
+    if (response.success) {
+      return {
+        status: 200,
+        data: response.data,
+      };
+    }
+    return {
+      status: 0,
+      data: undefined,
+    };
+  }
+
+  async function doDeleteSingleApiKey(
```

Comment:
`doDeleteSingleServiceDoc`?

Reply:
Good catch! I did a bit of copy&pasting between the two services. Looks like I missed renaming it 😄

Reply:
Done 👍